acq_stat_reports/acq_stat_reports.py

```diff
@@ -401,56 +406,61 @@ def acq_fails(acqs, range_datestart, range_datestop, outdir="out"):  #  noqa: AR
     little ones (by mag bin).
     """
 
-    fails = []
+    fails = {}
     range_acqs = acqs[
         (acqs["tstart"] >= range_datestart.cxcsec)
         & (acqs["tstart"] < range_datestop.cxcsec)
     ]
 
-    all_fails = range_acqs[range_acqs["acqid"] == 0]
-    failed_stars = []
-    for fail in all_fails:
-        star = {
-            "id": fail["agasc_id"],
-            "obsid": fail["obsid"],
-            "mag_exp": fail["mag"],
-            "mag_obs": fail["mag_obs"],
+    failed_stars = [
+        {
+            "id": int(mfail["agasc_id"]),
+            "obsid": int(mfail["obsid"]),
+            "mag_exp": mfail["mag"],
+            "mag_obs": mfail["mag_obs"],
             "acq_stat": "NOID",
         }
-        failed_stars.append(star)
-    make_fail_html(failed_stars, outdir / "failed_acq_stars_list.html")
+        for mfail in range_acqs[range_acqs["acqid"] == 0]
+    ]
+    n_stars = len(range_acqs)
+    n_failed = len(np.flatnonzero(range_acqs["acqid"] == 0))
+    fails[""] = {
+        # "mag_range": (-np.inf, np.inf),
```

Comment:
If this stays as a comment, maybe more context?  It isn't helping me at this point.